components/onedesk/actions/create-message/create-message.mjs

```diff
@@ -0,0 +1,44 @@
+import onedesk from "../../onedesk.app.mjs";
+
+export default {
+  key: "onedesk-create-message",
+  name: "Create Message",
+  description: "Creates a message or comment. [See the docs](https://www.onedesk.com/developers/#_create_comment)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    onedesk,
+    itemId: {
+      propDefinition: [
+        onedesk,
+        "itemId",
+      ],
+    },
+    description: {
+      type: "string",
+      label: "Description",
+      description: "Content of the message",
+    },
+    postType: {
+      propDefinition: [
+        onedesk,
+        "postType",
+      ],
+    },
+  },
+  async run({ $ }) {
+    const { data } = await this.onedesk.createComment({
+      data: {
+        itemId: this.itemId,
+        description: this.description,
+        postType: this.postType,
+        token: this.onedesk._authToken(),
```

Comment:
Pretty awkward that this endpoint uses a different name for the token than the others, but I saw in their docs that it indeed expects the params this way